src/Blocks/BaseBlock.php

```diff
@@ -33,11 +33,8 @@ abstract public function prepare_data(array $fields): array;
      */
     public function render(array $attributes)
     {
-
         $data = $this->prepare_data($attributes);
 
-        \Timber::$locations = '/block_templates/blocks';
```

Comment:
Since it's been added [here](https://github.com/greenpeace/planet4-master-theme/pull/2183/files#diff-d5a71432d6e94abc1260a448df306d8dfc76d15fa4047646a4d133c38cc131d7R84)

Reply:
Also the `block_templates` does not exist on the theme repo. Only in the plugin